src/main/java/com/company/ScrambleStrings.java

```diff
@@ -0,0 +1,62 @@
+package com.company;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ScrambleStrings {
+    public static List<String> scrambleStrings(int left, int right, List<String> strings) {
+        List<String> result = new ArrayList<>();
+        int j, length;
+        char letter;
+
+        if (right < 0 || left < 0)
+            throw new IllegalArgumentException("One of the interval bounds is negative.");
+
+        if (right <= left)
+            throw new IllegalArgumentException("Right interval bound is less than equal to the left one.");
+
+        if (strings == null)
+            throw new NullPointerException("String list is null.");
```

Comment:
You should throw IllegalArgumentException here instead. NullPointerException shouldn't be thrown programatically, because it's usually thrown by the code itself, then it becomes hard to distinguish https://stackoverflow.com/questions/3322638/is-it-okay-to-throw-nullpointerexception-programmatically#3323006